test/adapters/level_zero/CMakeLists.txt

```diff
@@ -44,3 +44,22 @@ if(LINUX)
 
     target_link_libraries(test-adapter-level_zero PRIVATE zeCallMap)
 endif()
+
+if(NOT WIN32)
+    # Make L0 use CallMap from a seprate shared lib so that we can access the map
+    # from the tests. This only seems to work on linux
+    add_library(zeCallMap SHARED zeCallMap.cpp)
+    target_compile_definitions(ur_adapter_level_zero PRIVATE UR_L0_CALL_COUNT_IN_TESTS)
```

Comment:
I think we should only call this add_library, target_compile_definitions and target_link_libraries once. Can we remove it from here or from under the if above?

Reply:
Right, but once my fix is merged this might stop working since we will call add_library(zeCallMap) twice. If this PR is merged first then I can rebase and extract this in my PR but otherwise this would need to be fixed here.